src/jit/compiler.cpp

```diff
@@ -9761,12 +9761,12 @@ int cTreeFlagsIR(Compiler* comp, GenTree* tree)
                 {
                     chars += printf("[CALL_HOISTABLE]");
                 }
-#ifdef LEGACY_BACKEND
+#ifdef GTF_CALL_REG_SAVE
```

Comment:
I don't think that you should have a flag value in an `#ifdef`. I would agree with Bruce's suggestion that you simply define it to be 0 for ARM.